pkg/cri/sbserver/service.go

```diff
@@ -349,8 +352,40 @@ func (c *criService) register(s *grpc.Server) error {
 
 // imageFSPath returns containerd image filesystem path.
 // Note that if containerd changes directory layout, we also needs to change this.
-func imageFSPath(rootDir, snapshotter string) string {
-	return filepath.Join(rootDir, plugin.SnapshotPlugin.String()+"."+snapshotter)
+func imageFSPath(rootDir, snapshotter string, client *containerd.Client) string {
```

Comment:
Would you be able to write a unit test for this?

Reply:
Possibly.. I'm not sure how I'd effectively use this in a unit test scenario though as it requires the daemon. Let me see if we have any CRI tests that write out a different config to use, I think that'd make the most sense